src/ui/SurveyUnit/Communication/CommunicationForm.tsx

```diff
@@ -0,0 +1,218 @@
+import Dialog from '@mui/material/Dialog';
+import { useMemo, useState } from 'react';
+import D from '../../../i18n/build-dictionary';
+import { surveyUnitIDBService } from '../../../utils/indexeddb/services/surveyUnit-idb-service';
+import {
+  communicationReasonEnum,
+  communicationStatusEnum,
+  communicationTypeEnum,
+} from '../../../utils/enum/CommunicationEnums';
+import CommunicationDialogContent from './CommunicationDialogContent';
+import CommunicationConfirmation from './CommunicationConfirmation';
+import { mediumRadioValues, reasonRadioValues, typeRadioValues } from '../../../utils/constants';
+
+enum Steps {
+  MEDIUM,
+  TYPE,
+  REASON,
+  VALIDATE,
+}
+
+namespace Steps {
+  export function after(value: Steps): Steps {
+    return value + 1;
+  }
+
+  export function before(value: Steps): Steps {
+    return value - 1;
+  }
+}
+
+export interface CommunicationRequestRadioData {
+  value: string;
+  label: string;
+  disabled: boolean;
+}
+
+export interface CommunicationRequestForm {
+  medium: string | undefined;
```

Comment:
```suggestion
  medium?: string;
```

Why not using the ? syntax ?